### ISSUE TITLE
cairocffi 0.7 cannot load gdk-pixbuf library

### ISSUE THREAD

**Issue by cdevienne** [SimonSapin/cairocffi#63]
with cairocffi 0.7, I get the following error:
```
In [1]: from cairocffi import pixbuf
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-1-502496f09f86> in <module>()
----> 1 from cairocffi import pixbuf

/home/cdevienne/.virtualenvs/unlish-dev/lib/python2.7/site-packages/cairocffi/pixbuf.py in <module>()
     28 
     29 gdk_pixbuf = dlopen(ffi, 'gdk_pixbuf-2.0', 'libgdk_pixbuf-2.0-0',
---> 30                     'libgdk_pixbuf-2.0.so')
     31 gobject = dlopen(ffi, 'gobject-2.0', 'libgobject-2.0-0', 'libgobject-2.0.so')
     32 glib = dlopen(ffi, 'glib-2.0', 'libglib-2.0-0', 'libglib-2.0.so')

/home/cdevienne/.virtualenvs/unlish-dev/lib/python2.7/site-packages/cairocffi/__init__.pyc in dlopen(ffi, *names)
     36             pass
     37     # Re-raise the exception.
---> 38     return ffi.dlopen(names[0])  # pragma: no cover
     39 
     40 

OSError: cannot load library 'gdk_pixbuf-2.0': gdk_pixbuf-2.0: cannot open shared object file: No such file or directory
```

The problem does not occur with cairocffi 0.6.

Digging a little, I found that it is related to the switch from cffi.FFI to _cffi_backend.FFI:
```
In [1]: import cffi

In [2]: import _cffi_backend

In [3]: cffi.FFI().dlopen('gdk_pixbuf-2')
Out[3]: <cffi.api.FFILibrary_gdk_pixbuf-2 at 0x7f893a299650>

In [4]: _cffi_backend.FFI().dlopen('gdk_pixbuf-2')                                                                              
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-4-eef9364611fa> in <module>()
----> 1 _cffi_backend.FFI().dlopen('gdk_pixbuf-2')

OSError: cannot load library 'gdk_pixbuf-2': gdk_pixbuf-2: cannot open shared object file: No such file or directory

```

That said, I have no clue why _cffi_backend is unable to load the library. I tried setting LD_LIBRARY_PATH, with no success.

**Comment by SimonSapin** [comment]
What’s the name of the `.so` file for PixBuf on your system? E.g. it’s `/usr/lib/libgdk_pixbuf-2.0.so` on mine.

**Comment by cdevienne** [comment]
on my ubuntu:
/usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0
/usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.3000.7

on the wheezy container I had the problem first:
/usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0
/usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.2600.1

**Comment by SimonSapin** [comment]
I have no clue, sorry. Maybe file an issue on CFFI?

**Comment by SimonSapin** [comment]
I just released v0.7.2 with this change: 729a5b8d348554bf3d0ccdc92406e1aaad50a7d9. Does it work for you?

**Comment by cdevienne** [comment]
Yes it does !

Thank you Simon :-)
